setup.py

```diff
@@ -4,7 +4,7 @@
 
 setup(
     name='tsd-file-api',
-    version='1.9.5',
+    version='1.9.6.dev1',
```

Comment:
My philosophy is that everything on master is ready for production, so I don't want versions tagged with `dev`. If you are still developing something, then keep it on a feature branch until it can be merged.

Reply:
OK. I can change it to 1.9.6

Reply:
Thanks! And also, thank you for the pull request. Great to get contributions :)

Reply:
In the master branch I can see many commits after the version is set to 1.9.5. In my mind a .dev release can be used for many commits, but a release number like 1.9.5 corresponds to one commit. For releases I would set it to a version number, create a tag and make a release.

Reply:
sure, tag and release is fine. but please remove the .dev1

Reply:
I will remove .dev1. My point is that now there are many versions of version 1.9.5, as there are many commits with that version number in setup.py. In my mind a release version number should only have one state.

Reply:
I understood your point.